lib/matplotlib/text.py

```diff
@@ -1147,9 +1147,10 @@ def set_verticalalignment(self, align):
         """
         Set the vertical alignment
 
-        ACCEPTS: [ 'center' | 'top' | 'bottom' | 'baseline' ]
+        ACCEPTS: [ 'center' | 'top' | 'bottom' | 'baseline' |
+                   'center_baseline' ]
         """
-        legal = ('top', 'bottom', 'center', 'baseline')
+        legal = ('top', 'bottom', 'center', 'baseline', 'centre_baseline')
```

Comment:
Typo.

Reply:
UK English, my UK fingers find it difficult to type center;)